main.tf

```diff
@@ -184,7 +185,7 @@ resource "aws_rds_cluster_instance" "cluster_instance_0" {
   db_parameter_group_name      = "${var.db_parameter_group_name}"
   preferred_maintenance_window = "${var.preferred_maintenance_window}"
   apply_immediately            = "${var.apply_immediately}"
-  monitoring_role_arn          = "${join("", aws_iam_role.rds-enhanced-monitoring.*.arn)}"
+  monitoring_role_arn          = "${coalesce("",join("", aws_iam_role.rds-enhanced-monitoring.*.arn))}"
```

Comment:
I don't understand how this change could fix anything. Coalesce will use the 2nd value here because the first value is always empty. If there is an issue with the 2nd value, the join (which looks correct to me), then I don't see how coalesce would make a difference.